### ISSUE TITLE
@import using relative paths

### ISSUE THREAD

**Issue by derwaldgeist** [fourseven/meteor-scss#145]
Since I upgraded to Meteor 1.2 and the latest fourseven:scss, it seems as if relative @import links aren't working anymore, even if the target file is in the same folder. Only a fully qualified path using {} at the beginning seems to work. Can you confirm this?

**Comment by sebakerckhof** [comment]
Relative imports should work in version 3.4.0-beta1

**Comment by derwaldgeist** [comment]
That's great to hear! Will try this version out.

**Comment by sebakerckhof** [comment]
Now that 3.4.1 is released. Can you confirm this works for you?

**Comment by derwaldgeist** [comment]
@sebakerckhof: I had to revert all my projects to 1.1, since I noticed very strange behaviours with 1.2 (too many packages were broken). But the most important "missing link" was ```fourseven:scss```. So, having the autoprefixer back and also the relative paths will enable me to try the upgrade out again. Did you also change the way absolute paths are working (i.e. to project-relative paths) in 3.4.1, since this had broken my Bower includes from the .meteor/build/bower folder? I remember you said you might change this.

**Comment by sebakerckhof** [comment]
That's not on trunk, but I have branch that handles urls starting with .meteor differently:
https://github.com/sebakerckhof/meteor-scss/tree/absolutes

But it needs more testing.

**Comment by derwaldgeist** [comment]
Hi @sebakerckhof: Is this absolute file path handling now enabled per default?

**Comment by chrise86** [comment]
> The docs also state it would be possible to put the SCSS file in a package instead. But how does this actually work? Do I have to state them in the package.js file?

Also interested to hear the answer to that one. Do you come up with an answer @derwaldgeist ?

**Comment by sebakerckhof** [comment]
@chrise86 Yes, you have to put them in your package.js file using `addFiles`.

**Comment by derwaldgeist** [comment]
@chrise86: Yes, I got it working now, but it caused me a lot of headaches. In the end, I am using a Meteor package for each bower component now, this was the only way I got ```mquandalle:bower``` to work with ```fourseven:scss``` again.

I also had to re-structure the way Bootstrap is included, because the SCSS in my packages in now loaded before the defaults in ```reywood:boostrap3-sass```, so I had to tweak this.

But there's also some benefits with this new configuration, as everything is even more modularized now. So I am closing this issue now.
